api/pkg/apis/v1alpha1/providers/target/kubectl/kubectl.go

```diff
@@ -165,25 +165,25 @@ func (i *KubectlTargetProvider) Init(config providers.IProviderConfig) error {
 		}
 	}
 	if err != nil {
-		sLog.Errorf("  P (Kubectl Target): %+v", err)
+		sLog.Errorf("  P (Kubectl Target): failed to get the cluster config: %+v", err)
 		return err
 	}
 
 	i.Client, err = kubernetes.NewForConfig(kConfig)
 	if err != nil {
-		sLog.Errorf("  P (Kubectl Target): %+v", err)
+		sLog.Errorf("  P (Kubectl Target): failed to create a new clientset: %+v", err)
 		return err
 	}
 
 	i.DynamicClient, err = dynamic.NewForConfig(kConfig)
 	if err != nil {
-		sLog.Errorf("  P (Kubectl Target): %+v", err)
+		sLog.Errorf("  P (Kubectl Target): failed to create a dynamic client: %+v", err)
 		return err
 	}
 
 	i.DiscoveryClient, err = discovery.NewDiscoveryClientForConfig(kConfig)
 	if err != nil {
-		sLog.Errorf("  P (Kubectl Target): %+v", err)
+		sLog.Errorf("  P (Kubectl Target): failed to create a discovery client: %+v", err)
```

Comment:
Trace ids?

Reply:
No trace Id for `Init()` method as above mentioned